packages/server/schema/schema.graphql

```diff
@@ -118,41 +179,63 @@ type CommentConnection {
   """
   totalCount: Int
 
-  """Offset from start"""
+  """
+  Offset from start
+  """
   startCursorOffset: Int!
 
-  """Offset till end"""
+  """
+  Offset till end
+  """
   endCursorOffset: Int!
 
-  """Information to aid in pagination."""
+  """
+  Information to aid in pagination.
+  """
   pageInfo: PageInfoExtended!
 
-  """A list of edges."""
+  """
+  A list of edges.
+  """
   edges: [CommentEdge]!
 }
 
-"""An edge in a connection."""
+"""
+An edge in a connection.
+"""
 type CommentEdge {
-  """The item at the end of the edge"""
+  """
+  The item at the end of the edge
+  """
   node: Comment
 
-  """A cursor for use in pagination"""
+  """
+  A cursor for use in pagination
+  """
   cursor: String!
 }
 
-"""Comment data"""
+"""
+Comment data
+"""
 type Comment implements Node {
-  """The ID of an object"""
+  """
+  The ID of an object
+  """
   id: ID!
 
-  """mongoose _id"""
+  """
+  mongoose _id
+  """
   _id: String!
   body: String
   user: User
   post: Post
   likesCount: Int!
 
-  """whether logged user liked this post"""
+  """
```

Comment:
why schema changed?

Reply:
My VSCode automatic Eslint made these changes.

Reply:
Also, I've created a query `searchPosts` to relay-compile get the types.

Reply:
you can regenerate using `yarn update-schema`

Reply:
Done! I forgot you create the schema via code. I use to change the `graphql.schema` directly.